lib/heroku/command/fork.rb

```diff
@@ -144,20 +144,22 @@ def migrate_db(from_addon, from, to_addon, to)
         to_config = api.get_config_vars(to).body
         to_attachment = to_addon["message"].match(/Attached as (\w+)_URL\n/)[1]
 
-        pgb = Heroku::Client::Pgbackups.new(from_config["PGBACKUPS_URL"])
-        transfer = pgb.create_transfer(
+        resolver = Heroku::Helpers::HerokuPostgresql::Resolver.new(to, api)
+        attachment = resolver.resolve("#{to_attachment}_URL", nil)
+        pgb = Heroku::Client::HerokuPostgresql.new(attachment)
+        transfer = pgb.pg_copy(
+          from_attachment.gsub('HEROKU_POSTGRESQL_',''),
           from_config["#{from_attachment}_URL"],
```

Comment:
So I'm not sure about the soundness of getting the URL like this, but you're not changing that aspect of this functionality, so I think it's fine for now--it's just something we may want to keep in mind, especially with the shareable add-on changes.